static/init.js

```diff
@@ -0,0 +1,14 @@
+// Creating a first, global "app" object so others
+// can use it before real app is actually created
+var app = {};
+
+
+// once the element is ready, transfer attributes
+function onAppReady(element) {
+    for (const key in app) {
+        element[key] = app[key];
+    }
```

Comment:
I do not remember exactly why, but if you use "for key in app" you have to check if the key exists in the dictionary.
if you use "for key of Object.keys(app)" you don't need it, probably because some keys can be "undefined" then you could do element[undefined] sometimes, but I am not 100% sure.
```suggestion
    for (const key of Object.keys(app)) {
        element[key] = app[key];
    }
```